### PR TITLE
Add Cubes OLAP container for dogstrust dataset.

### PR DESCRIPTION
**Pull request by daithiocrualaoich**
2af8bc4a643

### CODE REVIEW

File: dat/cubes.acbuild.sh [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+acbuild run -- apt-get install -qq python-pip
+acbuild run -- pip install -q --upgrade pip
+
+
+################################################################################
+# Cubes
+################################################################################
+
+acbuild run -- pip install click flask sqlalchemy
+acbuild run -- pip install cubes
```
Comment (passy) [inline dat/cubes.acbuild.sh]: Just curious: Why the split here?

File: dat/cubes.acbuild.sh [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+################################################################################
+# Basic Development Tools
+################################################################################
+
+acbuild run -- apt-get update -qq
+acbuild run -- apt-get upgrade -qq
+
+acbuild run -- apt-get install -qq wget
+acbuild run -- apt-get install -qq build-essential gcc
```
Comment (passy) [inline dat/cubes.acbuild.sh]: Is `gcc` not part of `build-essential`?

File: docs/dogstrust.rst [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+- ``needs_training BOOLEAN``: "Needs some training"
+- ``not_based_at_centre BOOLEAN``: "Living off site"
+- ``small_sparky BOOLEAN``: "Small but sparky"
+- ``stranger_friendly BOOLEAN``: "Stranger friendly"
+- ``young_at_heart BOOLEAN``: "Young at heart"
+
+
+OLAP
+~~~~
+Cubes_ is an open source OLAP_ server written in Python.
```
Comment (passy) [inline docs/dogstrust.rst]: How did I not know about this? Looks amazing!

File: dat/cubes.acbuild.sh (hunk 1 above)
Comment (daithiocrualaoich) [inline dat/cubes.acbuild.sh]: It's nothing very consistent. Here `cubes` was the  thing we actually wanted. The rest were dependencies we needed to get `cubes` to run, is all.

File: dat/cubes.acbuild.sh (hunk 2 above)
Comment (daithiocrualaoich) [inline dat/cubes.acbuild.sh]: Ah... thanks. Yes it is and I'd taken it out of some of the other containers. I'll make it fixed for good.

**Pull request merged**